### ISSUE TITLE
[FEATURE] : Make a contributors section

### ISSUE THREAD

**Issue by PritamSarbajna** [PritamSarbajna/tourism-website#18]
We can make a contributors section to list all the contributors for this project

**Comment by Joshua-Nweze** [comment]
hi, I'll love to do this one

**Comment by PritamSarbajna** [comment]
Yeah sure go for it. If you face any difficulty or need any idea do let me know here #17

**Comment by prakhartiwari2507** [comment]
Can you share some ideas of what that section should look like?

**Comment by Joshua-Nweze** [comment]
@PritamSarbajna I noticed you didn't use bootstrap in the project. Is it okay if i use it, with it we can make the site responsive

**Comment by PritamSarbajna** [comment]
> @PritamSarbajna I noticed you didn't use bootstrap in the project. Is it okay if i use it, with it we can make the site responsive

Hey @Joshua-Nweze don't use bootstrap or any other framework in this project that way this project will be still beginner friendly. You can learn from youtube and try to implement on your own .

**Comment by PritamSarbajna** [comment]
> Can you share some ideas of what that section should look like?

Yeah sure @prakhartiwari2507 just let me make a rough layout

**Comment by Joshua-Nweze** [comment]
@PritamSarbajna Okay, I won't use bootstrap

**Comment by prakhartiwari0** [comment]
Shouldn't this issue be closed? @PritamSarbajna

**Comment by PritamSarbajna** [comment]
@prakhartiwari0 Let me update the issue heading

**Comment by prakhartiwari0** [comment]
Is this project registered for hacktoberfest? @PritamSarbajna

**Comment by PritamSarbajna** [comment]
@prakhartiwari0 Nah man unfortunately no :(

**Comment by constneophyte** [comment]
Can i work on this issue?

**Comment by PritamSarbajna** [comment]
@constneophyte yeah sure go ahead

**Comment by the-amazing-atharva** [comment]
@PritamSarbajna Sorry for tagging you directly

I see this issue is unresolved.
Can I work on it?

**Comment by PritamSarbajna** [comment]
@the-amazing-atharva absolutely go for it 🚀

**Comment by the-amazing-atharva** [comment]
Thanks!
Working on it

**Comment by Shreyashkshirsagar** [comment]
Your Repository is a Huge success ! Very Hearty Congratulations to you :) Well The Ratio of actual contributors and the contributors on the Web Application really have a very drastic drop , So I would love to make things Right and add all the missing contributor to your Web-Page. Please Grant the permission to do so !
